extensions/amp-brightcove/1.0/component.js

```diff
@@ -164,3 +161,7 @@ function checkProps({account}) {
   }
   return true;
 }
+
+const BentoBrightcove = forwardRef(BentoBrightcoveWithRef);
+BentoBrightcove.displayName = 'BentoBrightcove'; // Make findable for tests.
+export {BentoBrightcove};
```

Comment:
We need to update the import in `base-element.js`

Reply:
Done